pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,49 +13,34 @@ node('jenkins-master') {
 }
 
 notify.wrap {
-  def retries = 3
   def repo = 'webserv/imgserv'
   def tag = 'dax_latest'
 
-  def run = {
-    dir('imgserv/lsst-dm-ci') {
+  node('docker') {
+    timeout(time: 1, unit: 'HOURS') {
+        git([
+          url: util.githubSlugToUrl('lsst/dax_imgserv'),
+          branch: 'master'
+        ])
+
         stage('build') {
-            retry(retries) {
-                build(repo)
-            }
+            withEnv(["DOCKER_REPO=$repo"]) {
+                util.bash './lsst-dm-ci/prod_image.sh'
+             }
         } // build
 
         stage('test') {
-            retry(retries) {
-                docker.image("${repo}:${tag}").inside {
-                    util.bash "./run_tests.sh"
-                }
+            docker.image("${repo}:${tag}").inside {
+                util.bash '/app/lsst-dm-ci/run_tests.sh'
             }
         } // test
 
         stage('publish') {
-            util.bash "./pub_image.sh"
+            docker.withRegistry('https://index.docker.io/v1/',
+            'dockerhub-sqreadmin') {
```

Comment:
I don't find this style of line breaking very attractive.  Maybe something like the below (if it won't fit on one line)?

```
docker.withRegistry(
  'https://index.docker.io/v1/',
  'dockerhub-sqreadmin'
)
```

Reply:
Changed as suggested.